mypyc/primitives/list_ops.py

```diff
@@ -83,7 +84,6 @@ def emit_new(emitter: EmitterInterface, args: List[str], dest: str) -> None:
     error_kind=ERR_FALSE,
     emit=call_emit('CPyList_SetItem'))
 
-
```

Comment:
I delete a blank line here and I find that one/two blank line(s) is used interchangeably. I am surprised that flake8 is OK with this